core/src/database/proxy_backend/session.cpp

```diff
@@ -63,8 +66,7 @@ proxy_statement_backend *proxy_session_backend::make_statement_backend() {
 }
 
 proxy_rowid_backend *proxy_session_backend::make_rowid_backend() {
-    // TODO: Create ROWID interface
-    return nullptr;
+    throw make_exception(api::ErrorCode::DATABASE_EXCEPTION , "ROWID is not supported by database backend");
```

Comment:
Why not sticking to the `nullptr`?

Reply:
Because I prefer that the day we use a rowid by mistake we have an explicit exception and not trying to find the reason of the failure. The fact that rowid is unsupported is documented in the database md file